codalab/worker/file_util.py

```diff
@@ -24,19 +29,28 @@ def tar_gzip_directory(
                    are not included.
     exclude_patterns: Any directory entries with the given names at any depth in
                       the directory structure are excluded.
+    ignore_file_name: Name of the file where exclusion patterns are read from.
```

Comment:
Would prefer `ignore_filename` or even just `ignore_file` to highlight the structure of the `ignore` part.